src/gocept/gtimelog/core.py

```diff
@@ -218,26 +218,27 @@ def totals(self, split_intern_customer=False):
 
     def icalendar(self, output):
         """Create an iCalendar file with activities."""
```

Comment:
What's that? I did not know that this is also a feature.

Reply:
I did not know either... let's keep it for now.